sources/osgViewer/input/InputConstants.js

```diff
@@ -0,0 +1,33 @@
+var Groups = {};
```

Comment:
This name is too generic IMO and leads to confusion, specially when it is spreaded in a lot of places

Reply:
Well it's exposed in OSG as InputGroups.
Then when it's used it always in correlation with the inputManager... it's defined in InputConstants... To me it seamed pretty clear.
Also I didn't want it to be too long because of method chaining. Prettier really like to break methods calls and I'm not so fond of it.

Reply:
Yeah but you use everywhere as Groups (https://github.com/cedricpinson/osgjs/pull/946/files#diff-b5e84f7f01f063c484d07c8e2385fa06R15). Maybe InputGroups could be better.